src/Microsoft.DotNet.Wpf/src/PresentationCore/ModuleInitializer.cs

```diff
@@ -19,6 +20,13 @@ public static void Initialize()
     {
         IsProcessDpiAware();
 
+        DWriteLoader.LoadDWrite();
+
+        AppDomain.CurrentDomain.ProcessExit += static (object sender, EventArgs e) =>
+        {
+            DWriteLoader.UnloadDWrite();
+        };
```

Comment:
Is this still required since AppDomain unloadability is no longer applicable? `AssemblyLoadContext.Unloading` could also be handled.

Reply:
I wrote this code by checking what `atexit` was mapped to in C#/IL when compiling C++/CLI to managed (It also added the same handler to `AppDomain.CurrentDomain.DomainUnload` but since there is only one AppDomain I didn't bother to add it). I doesn't seem useful to me but I decided to add it since it seemed harmless to me and I wanted to stay as close as possible to the C++/CLI code.